pkg/sharedconf/sharedconf.go

```diff
@@ -35,6 +35,17 @@ func Decode(v interface{}) error {
 		return err
 	}
 
+	// add some defaults
+	if sharedConf.GatewaySVC == "" {
+		sharedConf.GatewaySVC = "0.0.0.0:19000"
+	}
+
+	// TODO(labkode): would be cool to autogenerate one secret and print
+	// it on init time.
```

Comment:
👍 for this approach. Some do it from the "deployment code" by passing an env.var (so it does not get stored in the logs if they might be public). e.g. the MariaDB root pass: https://github.com/bitnami/bitnami-docker-mariadb/blob/master/README.md#setting-the-root-password-on-first-run